app/lib/rocketchat.js

```diff
@@ -1109,7 +1109,7 @@ const RocketChat = {
 		return ((room.prid || useRealName) && room.fname) || room.name;
 	},
 	getRoomAvatar(room) {
-		return room.prid ? room.fname : room.name;
+		return room.prid ? room.topic : room.name;
```

Comment:
Why topic? It work in some cases, but not in all of them.

Reply:
I will fix it by tomorrow

Reply:
It's wrong, if you open a discussion by some direct message room you'll see that image is incorrect because topic is `null`.

Reply:
Ok @djorkaeffalexandre I will correct it